pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -117,7 +117,11 @@ func generateRegistry(t *testing.T) string {
 
 	var buff bytes.Buffer
 
-	buff.WriteString(`// NOTE: This file is autogenerated
+	buff.WriteString(`// NOTE: This file was auto generated.  DO NOT EDIT DIRECTLY!
```

Comment:
The generated Typescript file has a header with instructions on how to run the generator, but the Go file does not.

I initially struggled figuring out how to run the generator, because I was looking in the Go file, which lacked the instructions. This change puts the instructions in both places.